static/app/components/onboarding/gettingStartedDoc/onboardingLayout.tsx

```diff
@@ -44,7 +44,9 @@ export type OnboardingLayoutProps = {
   projectSlug: Project['slug'];
   activeProductSelection?: ProductSolution[];
   configType?: ConfigType;
+  integration?: string;
   newOrg?: boolean;
+  provider?: string;
```

Comment:
I you decide to re-use it, I would extend it to not add unused props to this component.
```ts
interface FeatureFlagOnboardingLayout extends OnboardingLayoutProps {
  integration?: string;
  provider?: string;
}
```